darts/tests/test_probabilistic_models.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+
+from .base_test_class import DartsBaseTestClass
+from ..utils import timeseries_generation as tg
+from ..metrics import mae
+from ..logging import get_logger
+from darts.models import ExponentialSmoothing, ARIMA
+from darts.models.forecasting_model import GlobalForecastingModel
+
+logger = get_logger(__name__)
+
+try:
+    from ..models import RNNModel, TCNModel
+    from darts.utils.likelihood_models import GaussianLikelihoodModel
+    TORCH_AVAILABLE = True
+except ImportError:
+    logger.warning('Torch not available. TCN tests will be skipped.')
+    TORCH_AVAILABLE = False
+
+models_cls_kwargs_errs = [
+    (ExponentialSmoothing, {}, 0.4),
+    (ARIMA, {'p': 1, 'd': 0, 'q': 1}, 0.17)
+]
+
+if TORCH_AVAILABLE:
+    models_cls_kwargs_errs = np.concatenate([models_cls_kwargs_errs, [
```

Comment:
Silly question, but why using `np.concatenate` here (instead of say, something like `models_cls_kwargs_errs += [...]` ?

Reply:
No good reason at all, I changed it!